package.json

```diff
@@ -1,49 +1,67 @@
 {
   "name": "valtio",
+  "type": "module",
```

Comment:
ugh, unfortunately, this is not acceptable. we try to keep cjs first. we plan esm first for valtio v3.

Reply:
This pr is for main, I can send another pr using commonjs to v2 branch (this PR should be closed)